docs/Radarr/Radarr-collection-of-custom-formats.md

```diff
@@ -47,33 +47,48 @@ We've made 3 guides related to this.
-|                                               |                                         |                                           | [Peacock TV](#pcok)        |
-|                                               |                                         |                                           | [Paramount+](#pmtp)        |
-|                                               |                                         |                                           | [Stan](#stan)              |
-|                                               |                                         |                                           | [TVer](#tver)              |
-|                                               |                                         |                                           | [U-NEXT](#u-next)          |
-
-------
+---
+
+| Movie Versions                                | Unwanted                                | HQ Release Groups                         | Placeholder |
+| --------------------------------------------- | --------------------------------------- | ----------------------------------------- | ----------- |
+| [4K Remaster](#4k-remaster)                   | [3D](#3d)                               | [Remux Tier 01](#remux-tier-01)           |             |
+| [Criterion Collection](#criterion-collection) | [BR-DISK](#br-disk)                     | [Remux Tier 02](#remux-tier-02)           |             |
+| [Hybrid](#hybrid)                             | [Extras](#extras)                       | [Remux Tier 03](#remux-tier-03)           |             |
+| [IMAX Enhanced](#imax-enhanced)               | [LQ](#lq)                               | [UHD Bluray Tier 01](#uhd-bluray-tier-01) |             |
+| [IMAX](#imax)                                 | [LQ (Release Title)](#lq-release-title) | [UHD Bluray Tier 02](#uhd-bluray-tier-02) |             |
+| [Masters of Cinema](#masters-of-cinema)       | [Upscaled](#upscaled)                   | [UHD Bluray Tier 03](#uhd-bluray-tier-03) |             |
+| [Open Matte](#open-matte)                     | [x265 (HD)](#x265-hd)                   | [HD Bluray Tier 01](#hd-bluray-tier-01)   |             |
+| [Remaster](#remaster)                         |                                         | [HD Bluray Tier 02](#hd-bluray-tier-02)   |             |
+| [Special Edition](#special-edition)           |                                         | [HD Bluray Tier 03](#hd-bluray-tier-03)   |             |
+| [Theatrical Cut](#theatrical-cut)             |                                         | [WEB Tier 01](#web-tier-01)               |             |
+| [Vinegar Syndrome](#vinegar-syndrome)         |                                         | [WEB Tier 02](#web-tier-02)               |             |
+|                                               |                                         | [WEB Tier 03](#web-tier-03)               |             |
```

Comment:
Placeholder column intentional?

Reply:
I guess this might be for streaming services until you decide on the name? Not sure why the services aren't listed though if it's just a naming issue for the column's title...

Reply:
> Placeholder column intentional?

for the time being it's intentional, and makes the tables the same size

Reply:
> I guess this might be for streaming services until you decide on the name? Not sure why the services aren't listed though if it's just a naming issue for the column's title...

what do you mean the services aren't listed ?

Reply:
As in you're going to put the streaming services back there when you decide how to split them up

Reply:
no they are already split up, but it could happen then we need a need column for a new one